app/controllers/equipment_listings_controller.rb

```diff
@@ -1,32 +1,32 @@
 class EquipmentListingsController < ApplicationController
   def index
-    @equipments = Equipment.all
```

Comment:
Thank you for updating the methods! 
The instance variables need to be updated but I think Payla proposed to do this in slack.